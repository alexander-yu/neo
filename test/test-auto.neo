void print_name(string s) {
    print("My name is ");
    println(s);
}

void main() {
    auto i = 17;
    auto a = {| print, print_name |};
    println(i);
    println(a);
}