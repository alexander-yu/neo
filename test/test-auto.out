17
{|function at 0x400b3f, function at 0x4009e0|}
